tests/unit/misc/test_editor.py

```diff
@@ -157,6 +157,24 @@ def test_double_edit(self, editor):
             editor.edit("")
 
 
+class TestLineColumnCalculation:
```

Comment:
No need for a class here if you just have a single test method - just make that a function instead, similar to `test_modify`.